ddsketch/ddsketch.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, alpha=None, bin_limit=None, min_value=None):
             alpha = DEFAULT_ALPHA
         if bin_limit is None or bin_limit < 0:
             bin_limit = DEFAULT_BIN_LIMIT
-        if min_value < 0:
+        if min_value is None:
```

Comment:
```suggestion
        if min_value is None or min_value < 0:
```

Reply:
Oops! In retrospect this was obvious, thanks for pointing out.